openedx/features/colaraz_features/constants.py

```diff
@@ -1,7 +1,12 @@
 """
 Contants used by colaraz features app.
 """
-from student.roles import CourseStaffRole, CourseInstructorRole, OrgRoleManagerRole, CourseCreatorRole
+from student.roles import (
+    CourseStaffRole,
+    CourseInstructorRole,
+    OrgRoleManagerRole,
```

Comment:
ambiguous name...can we use something else ? like OrgManagerRole ?

Reply:
It's a role that manages organizational roles. The user who is assigned this role will be shown course access roles management panel. So kindly suggest a name accordingly.